src/components/Clock/Clock.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+class Clock extends React.Component {
+  state = {
+    clockName: this.props.name,
+    date: new Date(),
+  }
+
+  componentDidMount() {
+    this.intervalId = setInterval(() => {
+      // eslint-disable-next-line
+      this.setState({ clockName: this.props.name });
+      this.setState({ date: new Date() });
```

Comment:
```suggestion
      this.setState({ clockName: this.props.name, date: new Date() });
```

but also update name is redundant

Reply:
Thanks for review! I need to be more carefully in the future)